src/taskProvider.ts

```diff
@@ -93,15 +93,6 @@ export class TaskProvider implements vscode.TaskProvider {
       state: "enabled",
       allSelectedBehavior: "executeForEveryTarget",
     },
-    {
-      group: "Build",
-      name: "Build with debug mode",
```

Comment:
@cedelavergne-ledger the tooltip of the `Build` task needs to be updated with the name of the selected build use case.
Currently it always displays `Build app in release mode`. 

We can also decide to have a static tooltip that just says "Build app" but it's kind of... not really a tip.

Reply:
I have changed it already, with a static tootip